src/worker/fetch-events.js

```diff
@@ -99,3 +103,37 @@ function _groupEventsById(events, headers) {
 function _getLatest(eventsById, headers) {
   return _.map(eventsById, events => _.maxBy(events, event => event[headers["Aikaleima"]]));
 }
+
+function _getCode(city, index, event, headers) {
+  switch (city) {
+    case "Otaniemi":
+      return `${ city }_${ event[headers["eventId"]] }`;
+    default:
+      return `${ city }_${ index }`;
+  }
+}
+
+function _getLocation(event, headers) {
+  const lat = event[headers["locationLat"]];
+  const lng = event[headers["locationLon"]];
+
+  return _.isFinite(lat) && _.isFinite(lng)
+      ? `${lng},${lat}`
+      : null;
+}
+
+function _getTimeStamp(event, headers, column) {
+  const dateString = event[headers[column]];
+
+  return moment(dateString, moment.ISO_8601).isValid()
```

Comment:
Are the timestamps in the sheet in UTC? Or do they contain the timezone information with them?